src/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <main>  
+        <header>
+            <h1>Leaderboard</h1>
+        </header>
+        <section class="board">
+            <div class="scores">
+               <header>
+                <span>Receint Scores</span> 
+                
+                <button> Refresh</button>
+               </header>
+                <table> 
+                                    <tr>
+                        <td>Name : 100</td>
+                    </tr>
+                    <tr>
+                        <td>Name : 20</td>
+                    </tr>
+                    <tr>
+                        <td>Name : 50</td>
+                    </tr>
+                    <tr>
+                        <td>Name : 50</td>
+                    </tr>
```

Comment:
*[OPTIONAL]*

- Please, kindly ensure you've populated the name and score dynamically using javascript

Reply:
@zieeco  call it from API and displaying it is not this project task